test/jdk/ProblemList-FIPS140_2.txt

```diff
@@ -1233,3 +1233,8 @@ java/foreign/TestFallbackLookup.java    https://github.ibm.com/runtimes/backlog/
 sun/security/krb5/auto/Cleaners.java    https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
 sun/security/krb5/auto/S4U2selfNotF.java    https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
 java/util/jar/JarFile/IgnoreUnrelatedSignatureFiles.java    https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
+
+# Exclude the below tests from sanity.openjdk according to the issues:
+# https://github.ibm.com/runtimes/backlog/issues/1089
+javax/security/auth/kerberos/StandardNames.java https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
+sun/security/krb5/auto/CaseSensitive.java https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
```

Comment:
Similar question here, it was excluded for all platforms as well. In addition, should this be excluded for `FIPS140_3_OpenJcePlus` weak profile?
https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/39815c5f0502a80862574531bf453e6695a1580b/test/jdk/ProblemList-FIPS140_3_OpenJCEPlusFIPS.FIPS140-3.txt#L367

Reply:
The Weakly-Enforced profile enables all providers; it places "OpenJCEPlusFIPS" first at the list. Algorithms like the P12 KeyStore are supported when using this profile. So, we don’t need to exclude them.